tests/integration/product_create.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test;
+use Test::More;
+
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+
+remove_all_products();
+wait_dynamic_front();
+my $ua = new_client();
+create_user($ua, {});
+
+my %args_ref = (
```

Comment:
As a naming convention, we use the _ref suffix to indicate that a variable is a reference.

If it's a hash, then you can do %args = ( code => '1234' );

If it's a reference, then you need $ instead of % and  {} instead of() : $args_ref = { code => '1234' }